src/openrct2-ui/windows/ShortcutKeys.cpp

```diff
@@ -336,6 +341,21 @@ class ShortcutKeysWindow final : public Window
         InitialiseList();
     }
 
+    void ResetAll()
+    {
```

Comment:
To me this function feels like it should be part of the shortcut manager but i guess it does also need the refresh bindings. Are there any external calls to refresh bindings? Could you have a look at if it would make sense to move this function.

Reply:
Only the keys from the tab of the window are reset, so it's tied to the window in that regard.
I do realize the prompt message isn't clear about that, so I'll change that one.

Reply:
okay, suggest renaming it to work in that its just for the current tab.

Reply:
Good idea, I renamed the method.